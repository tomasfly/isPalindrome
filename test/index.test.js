const index = require('../src/index')

describe('index', () => {
    test('test duplicates', () => {
        const enteros = [3, 4, 5, 2, 2, 4, 5]
        const res = index.obtenerDuplicado(enteros)
        expect(res).toMatchSnapshot()
    });

    test('test palindrome', () => {
        const palabra = 'arepera'
        const res = index.isPalindrome(palabra)
        expect(res).toBe(true)
    });

    test('test palindrome falsecase', () => {
        const palabra = 'arbol'
        const res = index.isPalindrome(palabra)
        expect(res).toBe(false)
    });
});