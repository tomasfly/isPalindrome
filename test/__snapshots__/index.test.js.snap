// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index test duplicates 1`] = `
Array [
  2,
  4,
  5,
]
`;
